libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/StreamingExtensions/IBotFrameworkStreamingExtensionsAdapter.cs

```diff
@@ -0,0 +1,339 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.WebApi.StreamingExtensions
+{
+    public interface IBotFrameworkStreamingExtensionsAdapter
```

Comment:
I don't see how this class is different from the .Net Core implementation. Could you explain?